src/main/java/com/exadel/discountwebapp/history/enums/HistoryEnum.java

```diff
@@ -0,0 +1,13 @@
+package com.exadel.discountwebapp.history.enums;
+
+public enum HistoryEnum {
```

Comment:
Let's find more descriptive name like AuditableEntity, or AuditableEntityType, or something else that can describle this enum

Reply:
Used Audit naming for all classes